### PR TITLE
Add allow-mock-accuracy parameter

### PR DESCRIPTION
**Pull request by seadowg**
Needed for getodk/collect#4768

#### Before submitting this PR, please make sure you have:
- [x] included test cases for core behavior and edge cases in `tests_v1`
- [x] run `nosetests` and verified all tests pass
- [x] run `black pyxform` to format code
- [x] verified that any code or assets from external sources are properly credited in comments

### CODE REVIEW

**Comment by lognaturel** [comment]
The required dependency update has been merged.

File: pyxform/tests_v1/test_allow_mock_accuracy.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    def test_geopoint(self):
+        self.assertPyxformXform(
+            name="data",
+            md="""
+            | survey |           |             |          |                          |
+            |        | type      | name        | label    | parameters               |
+            |        | geopoint  | geopoint    | Geopoint | allow-mock-accuracy=true |
+            """,
+            xml__contains=[
+                'xmlns:orx="http://openrosa.org/xforms"',
```
Comment (lognaturel) [inline pyxform/tests_v1/test_allow_mock_accuracy.py:15]: This should be testing for the `http://www.opendatakit.org/xforms` namespace with `odk` prefix.

**Review by lognaturel — CHANGES_REQUESTED** [review]

File: pyxform/tests_v1/test_allow_mock_accuracy.py (hunk 1 above)
Comment (seadowg) [reply pyxform/tests_v1/test_allow_mock_accuracy.py:15]: Oh I think this is a mistake in `test_audio_quality` as well right (which I'm pretty sure is where I was copying from). That's probably come from somewhere else then got missed. So these tests should be matching on `'xmlns:odk="http://www.opendatakit.org/xforms"'` instead right?

**Review by lognaturel — APPROVED** [review]

File: pyxform/tests_v1/test_allow_mock_accuracy.py (hunk 1 above)
Comment (lognaturel) [reply pyxform/tests_v1/test_allow_mock_accuracy.py]: Whoops, good call fixing those.

**Pull request merged**